18-additional-resources.Rmd

```diff
@@ -76,15 +76,17 @@ Liu, M. C., & Huang, Y. M. (2017). The use of data science for education: The ca
 
 Rosenberg, J. M., Lawson, M. A., Anderson, D. J., Rutherford, T., & Jones, R. S. (accepted pending minor revisions). Making Data Science “Count”: Data Science and Learning, Design, and Technology Research. In E. Romero-Hall (Ed.), Research Methods in Learning Design & Technology. Routledge: New York, NY.
 
+Dutt, A., Ismail, M. A., & Herawan, T. (2017). A systematic review on educational data mining. IEEE Access, 5, 15991-16005.
```

Comment:
not only would I like to keep this, I'm looking forward to reading this! Thanks for adding this. I'm sorry that we missed it the first time around.

Reply:
its indeed very kind of you to say so. Thank you for adding it.